text2sql-backend/main.py

```diff
@@ -90,7 +90,7 @@ async def catch_exceptions_middleware(
         return JSONResponse({"status": "error", "message": str(e)})
 
 
-app.middleware("http")(catch_exceptions_middleware)
+# app.middleware("http")(catch_exceptions_middleware)
```

Comment:
Why remove this?

Reply:
Because it returns a 200 status code instead of 4XX, so axios wouldn't raise an exception and we wouldn't catch it.We'd have to go back to checking things twice: if the "status" is not "ok" and catching exceptions, instead of just catching exceptions...